suc/pkg/service/config.go

```diff
@@ -0,0 +1,137 @@
+package service
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/rancher/wins/cmd/server/config"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	ConfigDirEnvVar      = envVar{varName: "wins_config_dir"}
+	DebugEnvVar          = envVar{varName: "debug"}
+	AgentStringTLSEnvVar = envVar{varName: "strict_verify"}
+)
+
+const (
+	defaultConfigFile = "C:/etc/rancher/wins/config"
```

Comment:
Why is `C` sometimes capitalized and sometimes not?

Reply:
Good catch, will default to lower case to match install script